server/src/config/client/src/pages/Contact.scss

```diff
@@ -9,26 +9,55 @@
       align-items: flex-start;
 
       .form-inputs {
-        margin-bottom: 8rem;
         width: 100%;
         font-size: 1.2rem;
+        border: none;
+        margin-bottom: 8rem;
 
         input[type=text], input[type=email] {
         background-color: $background;
         color: $secondaryText;
-        }
+        border: solid 0.2rem $secondaryText;
+        border-radius: 0.5rem;
+      }
```

Comment:
Try not to split the bass css. keep the nested css after the base css of a section for easier readability